source/openapi-admin-v3.yaml

```diff
@@ -3064,6 +3064,201 @@ paths:
               schema:
                 properties: {}
 
+  /groups/{groupId}/apps/{appId}/graphql:
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+    post:
+      tags:
+        - graphql
+      operationId: adminRunGraphQLOperation
+      summary: Run a query or mutation
+      description: |-
+        Run a query or mutation as a system user that bypasses
+        authentication and data access rules. You can access metadata
+        about your API, including its schema, with an
+        [introspection](https://graphql.org/learn/introspection/) query.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required: ["query"]
+              properties:
+                query:
+                  type: string
+                  description: |-
+                    A GraphQL file string that contains one or more
```

Comment:
I'm unfamiliar with what you mean by 'GraphQL file string'. could you clarify the name here or link to some documentation explaining?